Formula/git-delta.rb

```diff
@@ -14,12 +14,23 @@ class GitDelta < Formula
   end
 
   depends_on "rust" => :build
-  uses_from_macos "llvm"
+  # LLVM is needed only at build time,
+  # don't install it on Linux if bottled.
+  depends_on "llvm" => :build unless OS.mac?
 
   conflicts_with "delta", :because => "both install a `delta` binary"
 
   def install
-    system "cargo", "install", "--root", prefix, "--path", "."
+    system "cargo", "build", "--release", "--locked"
+    bin.install "target/release/delta"
+  end
+
+  def caveats; <<~EOS
+    To configure git to use delta, run:
```

Comment:
This doesn't look like a [Homebrew-specific caveat](https://docs.brew.sh/Formula-Cookbook#caveats)?

Reply:
Yep, it was done by the request of project author: https://github.com/dandavison/delta/issues/24#issuecomment-541764922

Reply:
`caveats` are not a replacement for project documentation

Reply:
Removed.